backend/open_webui/utils/telemetry/logs.py

```
@@ -0,0 +1,53 @@
+import logging
```

Ruff (0.12.2)

`logging` imported but unused

Remove unused import: `logging`

(F401)